### PR TITLE
Send erros to jitsu

### PR DESCRIPTION
**Pull request by soutofernando**

### CODE REVIEW

File: components/LiveAnalytics.tsx [hunk 1]
```diff
@@ -39,10 +52,38 @@ if (document.readyState === 'complete') {
 
 type Props = Partial<Page> & { flags?: Flags };
 
+const IS_TESTING_JITSU = true;
```
Comment (lucis) [inline components/LiveAnalytics.tsx:55]: Remove

File: components/LiveAnalytics.tsx [hunk 2]
```diff
@@ -39,10 +52,38 @@ if (document.readyState === 'complete') {
...
+
+// Get all the scripts and check which ones have errors
+const errorHandlingScript = `
+window.__decoLoadingErrors = []
+            const scripts = document.querySelectorAll("script");
+            scripts.forEach((e) => e.onerror = () => __decoLoadingErrors.push(e.src))
+`;
```
Comment (lucis) [inline components/LiveAnalytics.tsx:62]: Better formatting

File: components/LiveAnalytics.tsx [hunk 3]
```diff
@@ -39,10 +52,38 @@ if (document.readyState === 'complete') {
...
+
+        {
+          /* 1. O Browser irá ler esse script e irá executar
+        2. Ele vai pegar todos os scripts q rodaram e vai armazenar na variável "scripts"
+        3. Depois ele vai dar uma passada nesse array de variaveis e as que tiverem erro, ele vai dar um push para o "__decoLoadingErrors"
+        4. Após tudo isso o jitsu ainda não está disponivel/ não carregou.
+        5. Com todo esse processo terminado, o script para enviar os erros para o jitsu é injetado na página, assim, o jitsu é acionado
+        6. __decoLoadingErrors.forEach Da uma passada no array e envia para o jitsu cada script que tiver com o erro dentro do array */
+        }
+
```
Comment (lucis) [inline components/LiveAnalytics.tsx:78]: Remove

**Review by lucis — CHANGES_REQUESTED** [review]

File: components/LiveAnalytics.tsx [hunk 4]
```diff
@@ -30,19 +39,38 @@ const init = async () => {
   onLCP(onWebVitalsReport);
 };
 
-if (document.readyState === 'complete') {
-  init();
-} else {
-  window.addEventListener('load', init);
+  window.onerror = function (message, url, lineNo, columnNo, error) {
```
Comment (igorbrasileiro) [inline components/LiveAnalytics.tsx:42]: What do you think about adding an event listener to the error event handler, something like this:
```js
window.addEventListener('error', function (...) {...}})
```

**Review by igorbrasileiro — APPROVED** [review]

**Pull request merged**